plugin.video.hrmagplayer/libplayer/hessenschau.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+class Hessenschau:
+    def getEpisodes(self, context, index, page):
+        context['episodes'] = list()
+        episode = None
+
+        article = self.getArticle(context, page)
+        ix = 0
+        while article != None:
```

Comment:
`and not xmbc.Monitor().abortRequested()` although I don't think this blocks for a long time. Note that when kodi requests your plugin to abort it only waits for 5 seconds before killing the script. If you have blocking loops that won't finish in 5 seconds this can be a problem